cli/main.go

```diff
@@ -277,6 +280,14 @@ func tabWriter() *tabwriter.Writer {
 	return tabwriter.NewWriter(os.Stdout, 1, 2, 2, ' ', 0)
 }
 
+func humanTime(ts *time.Time) string {
+	var fmted string
+	if !ts.IsZero() {
```

Comment:
I believe this should be inverted, the exceptional path should be indented, so `if ts.IsZero() { return "" }`